react-js/react-components-and-hierarchies.md

```diff
@@ -253,7 +253,7 @@ class App extends React.Component{
 
 export default App;
 ```
-[Run code](https://codepen.io/annu12340/pen/QWjpMRK?editors=1010)
+[Run code](https://codepen.io/annu12340/pen/QWjqrwO)
```

Comment:
@vegetabill said that `There is no App.js file in a CodePen so please revise.` but I don see any changed for that.